evalml/tests/automl_tests/test_automl.py

```diff
@@ -3452,7 +3452,8 @@ def test_automl_validates_problem_configuration(X_y_binary):
             problem_type="time series regression",
             problem_configuration={"max_delay": 2, "gap": 3},
         )
-
+    _, y = ts_data
+    X = pd.DataFrame(pd.date_range("2020-10-01", "2020-10-31"), columns=["Date"])
```

Comment:
Why did we need to add this to the test here?

Reply:
Originally I had added it because the code I included in `AutoMLSearch` would throw an error if the data didn't have a `date_index` column specified, which `X_y_binary` didn't have. Decided to just clean up the test and replace the data with what I added, thanks!